handlers/roll.py

```diff
@@ -4,19 +4,20 @@
 
 class Handler(MessageHandler):
     def __init__(self):
+        super().__init__()
         self.signal = "!roll"
 
         # displayed when !help is called
         self.description = self.signal + " <X>d<Y> : Rolls a Y-sided die X times."""
 
         # displatyed when !help test is called
         self.help = self.signal + """ <X>d<Y> : Rolls a Y-sided die X times."""
-        super().__init__()
 
 
     async def handle_message(self, client, message):
         args = message.content.split(" ", 1)
         if args[0] == self.signal:
+            print("got signal")
```

Comment:
If you would remove this that would be great. Thanks :D

Reply:
done